modelling/src/test/java/org/axonframework/modelling/command/AggregateAnnotationCommandHandlerTest.java

```diff
@@ -170,7 +172,9 @@ void testSupportedCommands() {
     void testCommandHandlerSubscribesToCommands() {
         verify(commandBus).subscribe(eq(CreateCommand.class.getName()),
                                      any(MessageHandler.class));
-        verify(commandBus).subscribe(eq(UpdateCommandWithAnnotatedMethod.class.getName()),
+        // Is subscribed two times because of the duplicate handler. This is good and indicates usage of the
+        // DuplicateCommandHandlerResolver
+        verify(commandBus, times(2)).subscribe(eq(UpdateCommandWithAnnotatedMethod.class.getName()),
```

Comment:
This test should be obsolete with the change in the `AnnotatedHandlerInspector`, right?

Reply:
What do you mean by obsolete? the `times(2)` change is valid, see the comment. I have added an additional handler, with a different signature. The change in `AnnotatedHandlerInspector` is to prevent abstract methods from being registered twice (for the same implementation)